jobs/sequence_v3/rc-4.yaml

```diff
@@ -0,0 +1,256 @@
+name: "sequence_v3/rc_4"
+version: "1"
+description: "Sequence v3 contracts (release candidate 3) (commit fa0fae3de26e189cf4f8a554b2d72f0a35297bc0) (evm paris)"
+depends_on: ["p256-verifier"]
+
+
+# Changelog since rc-3
+#
+# - ERC4337FactoryWrapper
+#   - Added new ERC4337FactoryWrapper contract wrapping Factory:
+#     - deploy() gated by a fixed senderCreator address.
+#     - Precomputes the CREATE2 wallet address and returns it if already deployed,
+#       otherwise forwards to Factory.deploy().
+
+# - ERC4337v07
+#   - validateUserOp() now calls isValidSignature(userOpHash, userOp.signature)
+#     directly instead of this.isValidSignature(...).
+#
+# - BaseAuth
+#   - Static-signature branch now sets _payload.noChainId from signatureFlag
+#     before computing opHash.
+#   - Updated BaseSig.recover(...) calls to use BaseSig.RecoverMode.Initial.
+#   - isValidSignature(bytes32,bytes) visibility changed from external to public.
+#
+# - BaseSig
+#   - Introduced RecoverMode enum (Initial / UseProvidedCheckpointer / SkipSnapshotRead).
+#   - recover(...) now takes RecoverMode instead of a boolean ignoreCheckpointer.
+#   - recoverChained(...) updated to pass the appropriate RecoverMode to nested recover() calls.
+#   - Added ChainedSignatureNestedInChainedSignature error and revert if a chained
+#     signature is nested inside another chained signature.
+#
+# - Sessions (SessionManager / SessionSig)
+#   - SessionSig.recoverSignature(...) now takes the wallet address as first parameter.
+#   - Per-call signature hash now uses:
+#     - payloadHash = Payload.hashFor(payload, wallet)
+#     - callHash = keccak256(abi.encodePacked(payloadHash, callIdx))
+#   - Exposed new helper hashPayloadCallIdx(wallet, payload, callIdx) and private
+#     _hashPayloadHashCallIdx(...); removed hashCallWithReplayProtection(...).
+#   - SessionManager now passes wallet = msg.sender into SessionSig.recoverSignature(...).
+#
+# - P256
+#   - Removed: P256VerificationFailed error declaration, N constant,
+#     verifySignatureAllowMalleability(...), normalized(...),
+#     tryDecodePoint(...), tryDecodePointCalldata(...).
+#   - Library API now consists of verifySignature(...) plus VERIFIER,
+#     RIP_PRECOMPILE and _HALF_N constants.
```

Comment:
If we are making changes to P256, we should be redeploying Passkeys. If not, remove this comment?